examples/features/nse-composition/README.md

```diff
@@ -39,8 +39,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: ${NAMESPACE}
 
-bases:
+resources:
 - config-file.yaml
+bases:
 - ../../../apps/nsc-kernel
 - ../../../apps/nse-kernel
 - ./passthrough-1
```

Comment:
It actually looks like there is no need for al of these subdirectories as you can import all of `passthrough-1`, ..., `nse-firewall` pod description files directly as resources.

Reply:
ok, done